relentless/optimize/method.py

```diff
@@ -280,57 +351,116 @@ def max_iter(self, value):
 class SteepestDescent(Optimizer):
     r"""Steepest descent algorithm.
 
-    For an :class:`~relentless.optimize.ObjectiveFunction` :math:`f\left(\mathbf{x}\right)`,
+    For an :class:`~relentless.optimize.objective.ObjectiveFunction` :math:`f\left(\mathbf{x}\right)`,
     the steepest descent algorithm seeks to approach a minimum of the function.
 
-    With an initial numerical guess for all the design variables :math:`\mathbf{x}`,
-    the following iterative calculation is performed:
+    :math:`\mathbf{X}=\left[X_1,\ldots,X_n\right]` is defined as the scaling
+    parameters for the gradient such that :math:`y_i=\frac{x_i}{X_i}`. By
+    default, the variables are left unscaled (:math:`X_i=1`).
+
+    With scaling, the gradient of the function becomes:
 
     .. math::
 
-        \mathbf{x}_{n+1} = \mathbf{x}_{n}-\alpha\nabla f\left(\mathbf{x}\right)
+        \nabla f\left(\mathbf{y}\right) = \left[{X_1}^2 \frac{\partial f}{\partial x_1},
+                                                      \cdots,
+                                                {X_n}^2 \frac{\partial f}{\partial x_n}\right]
```

Comment:
```suggestion
        \nabla f\left(\mathbf{y}\right) = \left[X_1 \frac{\partial f}{\partial x_1},
                                                      \cdots,
                                                X_n \frac{\partial f}{\partial x_n}\right]
```

Just one factor of `X_i` for the gradient. You don't get the second factor of `X_i` until the update equation (that one coming from the conversion between `x_i` and `y_i`.)